models/dex/solana/dex_solana_price_hour.sql

```diff
@@ -0,0 +1,70 @@
+{{
+  config(
+        schema = 'dex_solana',
+        alias = 'price_hour',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.hour')],
+        unique_key = ['blockchain', 'contract_address', 'hour'],
+        pre_hook='{{ enforce_join_distribution("PARTITIONED") }}',
+        post_hook='{{ expose_spells(\'["solana"]\',
+                            "project",
+                            "dex",
+                            \'["get_nimbus"]\') }}')
+}}
+
+{ % set project_start_date = '2022-03-10' % } --grabbed min block time from whirlpool_solana.whirlpool_call_swap #}
```

Comment:
Here is a syntax typo:
```suggestion
{% set project_start_date = '2022-03-10' %} --grabbed min block time from whirlpool_solana.whirlpool_call_swap
```